_action_files/nb2post.py

```diff
@@ -13,25 +13,12 @@ def _nb2htmlfname(nb_path, dest=None):
     if dest is None: dest = Config().doc_path
     return Path(dest)/fname
 
-# Add embedded links for youtube and twitter
-def add_embedded_links(cell):
-    "Convert block quotes to embedded links in `cell`"
-    _styles = ['youtube', 'twitter']
-    def _inner(m):
-        title,text = m.groups()
-        if title.lower() not in _styles: return f"> {m.groups()[0]}: {m.groups()[1]}"
-        return '{% include '+title.lower()+".html content=\'`"+_to_html(text)+"`\' %}"
```

Comment:
See how to_html is wrapped with backticks?  This is important, this is not something that comes with nbdev and why we are forced to monkey patch here.